package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@blackbaud/skyux-lib-help",
-  "version": "1.3.1",
+  "version": "2.0.1",
```

Comment:
This should be `2.0.0`.

Reply:
Ha, that's what I was going to do, but learned that our VSTS CI build started with 2.0.1 so I was copying that.  I'll update it.